src/partition/ReceivedPartition.cpp

```diff
@@ -426,11 +420,10 @@ void ReceivedPartition::compareBoundingBoxes()
 
     // send connectionMap to other master
     m2n().getMasterCommunication()->send(connectedRanksList, 0);
-    if (not connectionMap.empty()) {
-      com::CommunicateBoundingBox(m2n().getMasterCommunication()).sendConnectionMap(connectionMap, 0);
-    } else {
-      PRECICE_ERROR("This participant seems to have no mesh partitions for mesh " << _mesh->getName() << " at the coupling interface.");
-    }
+    PRECICE_CHECK(not connectionMap.empty(), "The mesh \"" << _mesh->getName() << "\" of this participant has no partitions at the coupling interface. "
```

Comment:
```suggestion
    PRECICE_CHECK(not connectionMap.empty(), "The mesh \"" << _mesh->getName() << "\" of this participant seems to have no partitions at the coupling interface. "
```
It depends who defines where the coupling interface is. This participant or the other one? I prefer the weaker statement here.

Reply:
This check is in the master block of the `ReceivedPartition`, so the side should be clear here.

Reply:
What I meant is: who defines where the coupling interface is? This participant or the other one? The error here means in the end that both sides probably defined different coupling interfaces (concerning the geometry). Maybe we should just reformulate the first sentence completely. 
"Re-partitioning the received mesh XXX resulted in no overlaps with provided meshes."
The suggestions afterwards are good.